K8s/helm/values.yaml

```diff
@@ -3,7 +3,7 @@
 image:
   repo: asserts
   version: latest
-  pullPolicy: Always
+  pullPolicy: IfNotPresent
```

Comment:
For `latest` will this ensure that the image hash between local cache and remote are compared? Since this is just a demo app, wondering if we really need to use versioning hence the question.